### PR TITLE
Change versions to fix memory.limit_in_bytes: device or resource busy

### PR DESCRIPTION
**Pull request by flouthoc**
Change Operator version to `to fix memory.limit_in_bytes: device or resource busy`
Issue is that the upgrade for docker to fix CVE-2019-5736 has required that runC be loaded into memory: lxc/lxc@6400238.
The data is about 12Mi in size, so it's too big to work with the 10Mi containers that Prometheus Operator creates for the configuration reloader side-cars.

Fixed in newer Versions of Prometheus Operators therefore changing versions.

References:
https://github.com/helm/charts/issues/11447
https://github.com/coreos/prometheus-operator/pull/2403

### CODE REVIEW

**Comment by flouthoc** [comment]
@chrisz100 `pull-kops-e2e-kubernetes-aws` seems to be getting timed-out due to already exists namespace error.

**Comment by flouthoc** [comment]
@chrisz100 Could you please take a look.

File: addons/prometheus-operator/v0.29.0.yaml [hunk 1]
```diff
@@ -0,0 +1,16021 @@
...
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
```
Comment (chrisz100) [inline addons/prometheus-operator/v0.29.0.yaml]: This is very permissive. Does prometheus really require all these things? (not the expert here)

File: addons/prometheus-operator/v0.29.0.yaml [hunk 2]
```diff
@@ -0,0 +1,16021 @@
...
+                        description: 'Number of seconds after which the probe times
+                          out. Defaults to 1 second. Minimum value is 1. More info:
+                          https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes'
+                        format: int32
+                        type: integer
+                  name:
+                    description: Name of the container specified as a DNS_LABEL. Each
+                      container in a pod must have a unique name (DNS_LABEL). Cannot
+                      be updated.
+
```
Comment (chrisz100) [inline addons/prometheus-operator/v0.29.0.yaml]: Very permissive again...

**Comment by chrisz100** [comment]
The Roles to be used are very permissive. Is this something Prometheus requires?

File: addons/prometheus-operator/v0.29.0.yaml (hunk 1 above)
Comment (flouthoc) [reply addons/prometheus-operator/v0.29.0.yaml]: @chrisz100 These are the permissions from previous manifests , I think its already there on previous versions as well.

**Pull request merged**